### DISCUSSION TITLE
Can't throw a CommandSyntaxException with a component

### DISCUSSION THREAD

**Discussion by GliczDev** [PaperMC/Paper#10747] [Ideas]
### Is your feature request related to a problem?

Brigadier API has been merged lately, however it's missing one important feature - CommandSyntaxExceptions with components.

### Describe the solution you'd like.

Maybe something like `ComponentMessage` class? 

### Describe alternatives you've considered.

`PaperAdventure#asVanilla` which returns nms component that implements brigadier `Message`.

**Comment by jpenilla** [comment]
Use the `MessageComponentSerializer`

**Reply by GliczDev** [reply]
Oh, okay, thanks
